### DISCUSSION TITLE
Best way to handle order of headers

### DISCUSSION THREAD

**Discussion by misuszan** [VerifyTests/Verify#697] [Q&A] [answered]
Hello,

I wrote a test for an HttpTrigger Azure Function and am using Verify to verify the `HttpResponseMessage` that's returned by the http client. When I run the test locally, `Content-Type` is before `Content-Length`, but when the test runs during a build pipeline the order is switched. I want to keep `Content-Type` in the verification becuase it must be in the headers but I cannot find anyway to influence the order of these two headers.

What is the best practice here? Is there a way to modify VerifierSettings to better fit this scenario? None of the `VerifierSettings` seem to get the behavior I want. eg) Ignoring `Content-Length` leaves a blank `{ }` before/after `Content-Type` so the test still passes locally/failes in build.

For context this is: `System.Net.Http.HttpResponseMessage`.`Content`.`Headers` (`System.Net.Http.Headers.HttpContentHeaders`)

Thank you in advanced

Diff shows verified/received from pipeline
![image](https://user-images.githubusercontent.com/43422360/199621848-8b0b3a6a-3478-43b0-af6d-feb983c1f44f.png)

**Comment by SimonCropp** [comment, marked as answer]
are you using https://github.com/VerifyTests/Verify.Http  ?

**Reply by misuszan** [reply]
I was not, I was using `VerifyTests` and `VerifyMSTest`. `Verify.Http` is working wonderfully thank you so much!

**Reply by SimonCropp** [reply]
good to hear. do you have any any outstanding issues?
